ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,548 @@
+    public function select_db(string $database);
+
+    /** @return bool */
+    public function set_charset(string $charset);
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function options(int $option, $value);
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function set_opt(int $option, $value);
+
+    /** @return bool */
+    public function ssl_set(
+        string $key,
+        string $cert,
+        string $certificate_authority,
+        string $certificate_authority_path,
+        string $cipher
+    );
+
+    /** @return string|false */
+    public function stat();
+
+    /** @return mysqli_stmt|false */
+    public function stmt_init();
+
+    /** @return mysqli_result|false */
+    public function store_result(int $flags = 0);
+
+    /** @return bool */
+    public function thread_safe();
+
+    /** @return mysqli_result|false */
+    public function use_result();
+
+    /** @return bool */
+    public function refresh(int $options);
+}
+
+class mysqli_warning
+{
+    /**
+     * object $mysqli_link
+     *
+     * @return object|false
+     */
+    public function __construct();
+
+    /** @return bool */
+    public function next();
+}
+
+class mysqli_result
+{
+     /**
+      * mysqli $mysqli_link,
+      * int $resmode = MYSQLI_STORE_RESULT
+      *
+      * @return object|false
+      */
+     public function __construct();
+
+    /** @return void */
+    public function close();
+
+    /** @return void */
+    public function free();
+
+    /** @return bool */
+    public function data_seek(int $offset);
+
+    /** @return object|false */
+    public function fetch_field();
+
+    /** @return array */
+    public function fetch_fields();
+
+    /** @return object|false */
+    public function fetch_field_direct(int $field_nr);
+
+    /** @return array|false */
+    public function fetch_all(int $result_type = MYSQLI_NUM);
+
+    /** @return array|null|false */
+    public function fetch_array(int $result_type = MYSQLI_BOTH);
+
+    /** @return array|null */
+    public function fetch_assoc();
+
+    /** @return object|null */
+    public function fetch_object(string $class_name = 'stdClass', array $params = []);
+
+    /** @return array|null */
+    public function fetch_row();
+
+    /** @return bool */
+    public function field_seek(int $field_nr);
+
+    /** @return void */
+    public function free_result();
+}
+
+class mysqli_stmt
+{
+    /**
+     * mysqli $mysqli_link
+     * string $statement
+     */
+    public function __construct();
+
+    /** @return int|false */
+    public function attr_get(int $attr);
+
+    /** @return bool */
+    public function attr_set(int $attr, int $mode_in);
+
+    /** @return bool */
+    public function bind_param(string $types, &...$vars);
+
+    /** @return bool */
+    public function bind_result(&...$vars);
+
+    /** @return bool */
+    public function close();
+
+    /** @return void|false */
```

Comment:
void can't be used together with any other type. You should use `null` instead

Reply:
Fixed!